python/cudf/cudf/tests/test_groupby.py

```diff
@@ -331,28 +314,91 @@ def emulate(df):
     assert_eq(expect, got)
 
 
-@pytest.mark.parametrize("nelem", [100, 500])
+@pytest.mark.parametrize("nelem", [2, 3, 100, 500, 1000])
 @pytest.mark.parametrize(
     "func",
     ["mean", "std", "var", "min", "max", "idxmin", "idxmax", "count", "sum"],
 )
-def test_groupby_cudf_2keys_agg(nelem, func):
-    got_df = (
-        make_frame(DataFrame, nelem=nelem)
+def test_groupby_2keys_agg(nelem, func):
+    # gdf (Note: lack of multiIndex)
+    expect_df = (
+        make_frame(pd.DataFrame, nelem=nelem)
         .groupby(["x", "y"], sort=True)
         .agg(func)
     )
-
-    # pandas
-    expect_df = (
-        make_frame(pd.DataFrame, nelem=nelem)
+    got_df = (
+        make_frame(DataFrame, nelem=nelem)
         .groupby(["x", "y"], sort=True)
         .agg(func)
     )
+
     check_dtype = False if func in _index_type_aggs else True
     assert_eq(got_df, expect_df, check_dtype=check_dtype)
 
 
+@pytest.mark.skipif(
+    rmm._cuda.gpu.runtimeGetVersion() < 11000,
+    reason="These aggregations are not supported on CUDA 10.x.",
+)
```

Comment:
Actually, instead of skipping these, can we have these throw if the CUDA version is unsupported? If someone is running on CUDA 10.x these APIs will return arbitrary garbage data silently, which is worse than throwing.